R/deprecated.R

```diff
@@ -1,20 +1,3 @@
-#' Deprecated `TealData` class and related functions
-#'
-#' @description
-#' `r lifecycle::badge("deprecated")`
-#'
-#' The `TealData` class and associated functions have been deprecated. Use [teal_data()] instead.
-#' See the [Migration guide](https://github.com/insightsengineering/teal/discussions/945) for details.
-#'
-#' @name TealData
-#'
-#' @param ... any argument supported in `TealData` related functions.
-#'
-#' @return nothing
-#' @seealso  [cdisc_data()] , [join_keys()]
-#'
-NULL
-
 .deprecate_function <- function(what, details) {
```

Comment:
Hey @llrs-roche since this function was used to deprecate things in `0.4.0` I think it also should be removed. I grepped to see for the occurrence of this function in the codebase and it looks like there is another function that uses that:

```r
#' @rdname col_labels
#' @include formatters_var_labels.R
#' @details
#' `r lifecycle::badge("deprecated")`
#'
#' In previous versions of `teal.data` labels were managed with `get_labels()`.
#' This function is deprecated as of `0.4.0`, use `col_labels` instead.
#' @export
get_labels <- function(...) {
  .deprecate_function("get_labels()", "Use col_labels(data)")
}
```

Please remove `.deprecate_function` and remove `get_labels`

Reply:
Ok, I'll remove both.